### PR TITLE
Use most recent SCSS

### PR DESCRIPTION
**Pull request by bonanitech**
The `forge-dark` theme is using an old version of the SCSS file. This PR updates it to the most recent version.

### CODE REVIEW

File: forge-dark/forge-dark-theme.scss [hunk 1]
```diff
@@ -289,9 +301,15 @@ $group-default-label-color: $primary-text-color;
 $tourGuide-border: #D12B2B; //#c56c6c; //<<
 $tourGuide-heading-color: #BC3838; //#c56c6c; //<<
 
+$grip-color: #ccc;
+
+$icons-flow-color: #808080;
+
+$spinner-color: #999;
+
```
[the remark this answers is not in the document]
Comment (Steve-Mcl) [reply forge-dark/forge-dark-theme.scss:309]: The greys are too grey ;) 

A slight blue tinge helps them blend in better.

Thank you for pointing out the missing vars

File: forge-dark/forge-dark-theme.scss [hunk 2]
```diff
@@ -207,20 +213,24 @@ $node-icon-background-color-opacity: 0.05;
 $node-icon-border-color: #000;
 $node-icon-border-color-opacity: 0.1;
 
+$node-config-background: #f3f3f3;
```
Comment (Steve-Mcl) [inline forge-dark/forge-dark-theme.scss:216]:
```suggestion
$node-config-background: #f1f2f9;
```

Comment (Steve-Mcl) [reply forge-dark/forge-dark-theme.scss:216]: A slight blue tinge helps it blend in better.

Thank you for pointing out the missing var

**Review by Steve-Mcl — COMMENTED** [review]
I think apart from the slight color tweaks - all good :) 

Thank you for this.

**Comment by bonanitech** [comment]
Please check the colors for diff before I commit the requested changes.

**Comment by Steve-Mcl** [comment]
The diff looks fine apart from the suggestions. 

You can merge the suggested changes directly in this page (press `commit suggestion`) or in your local repo.

**Pull request merged**